app/assets/stylesheets/email.scss

```diff
@@ -29,7 +29,7 @@ a.underline-hover:hover { text-decoration: underline; }
 
 .callout { display:inline-block; padding-right:4px; padding-left:4px; line-height: 20px; background-color: #feffbc; text-decoration: none; }
 
-.h1,.h2,.h3,.h4,.h5,.h6,.h7 { font-family: "Helvetica Neue", Helvetica, Arial, "Lucida Grande", sans-serif; font-weight:normal; margin-bottom:15px; color:#000; }
+.h1,.h2,.h3,.h4,.h5,.h6 { font-family: "Helvetica Neue", Helvetica, Arial, "Lucida Grande", sans-serif; font-weight:normal; margin-bottom:15px; color:#000; }
```

Comment:
Color literals like `#000` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Properties should be ordered color, font-family, font-weight, margin-bottom<br>Each selector in a comma sequence should be on its own line<br>Colon after property should be followed by one space